tools/debugging/replay_wal.py

```diff
@@ -167,18 +228,25 @@ def replay_wal(
             to_canonical_address(partner_address),
         )
 
-        if not channel_state:
+        if channel_state is None:
             continue
 
         ###
         # Customize this to filter things further somewhere around here.
         # An example would be to add `import pdb; pdb.set_trace()`
         # and inspect the state.
         ###
-
         print_state_change(state_change, translator=translator)
         print_events(chain.from_iterable(events), translator=translator)
 
+        # Enable to print color coded presence state of channel partners
+        # print_presence_view(chain_state, translator)
+
+        # Enable to print balances & balance sum with all channel partners
+        # print_node_balances(chain_state, token_network_address, translator)
+
+        print_nl()
```

Comment:
shouldn't `print_nl` also be commented out?

Reply:
I found, that it is really helpful to have a visual guide to each state change context, so I figured it does not hurt.